.gitignore

```diff
@@ -24,3 +24,4 @@ yarn-debug.log*
 yarn-error.log*
 
 package-lock.json
+yarn.lock
```

Comment:
`yarn.lock` shouldn't be ignored
https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored